src/main/java/duke/pack/Command.java

```diff
@@ -0,0 +1,14 @@
+package duke.pack;
+
+/**
+ * represents a command
+ */
+public abstract class Command {
```

Comment:
Hmm, is there a need for an abstract class, would an interface suffice? Unless your planning to add a working function under commands